src/components/CocktailDetail/index.jsx

```diff
@@ -3,6 +3,8 @@ import axios from 'axios';
 import './style.css';
 import { useParams } from 'react-router-dom';
 import Carousel from '../Carousel';
+import PropTypes from 'prop-types';
+
 
```

Comment:
(Almost sure that Prettier/ESLint forbid you having 2 successive empty lines... I iz suspicious, have you set it up ?)